app/controllers/admin/subcategories_controller.rb

```diff
@@ -0,0 +1,70 @@
+module Admin
+  class SubcategoriesController < AdminController
```

Comment:
what about using single controller for `Categories, Subcategories, Teams`?

Reply:
I tried to do last week but couldn't manage to wrap my head around it... Maybe we can discuss it? How do you define a a general Model to call its methods?